pytroll_collectors/segments.py

```diff
@@ -491,6 +535,23 @@ def _find_time_slot(self, time_obj):
 
         return str(time_obj)
 
+    def check_schedule_time(self, check_time, raw_hour, raw_minute):
```

Comment:
I think it would be better to pass a python time object instead of hour+minute. That limits the number of arguments to the function, and won't break backwards compatibility if we decide to add eg seconds in the future.